### DISCUSSION TITLE
Recognize PostgreSQL Views for GraphQL queries

### DISCUSSION THREAD

**Discussion by gnadenthal** [keystonejs/keystone#7634] [Feature requests]
I have several custom views in my PostgreSQL db (using various subqueries and window functions that I think would be prohibitive or impossible to do in GraphQL using just the autogenerated table schemas). I would like keystone to recognize the view definitions (names and columns) and autogenerate GraphQL query and aggregate query access through its GraphQL endpoint. Mutations could be ignored. All that would be needed for views is query access.  I think this would be a great advantage in keystone. Thanks for considering.

**Comment by Achisingh** [comment]
Hi @gnadenthal we are currently restricted by Prisma on this, you can track their support for it at https://github.com/prisma/prisma/issues/678. If and when Prisma support it we will review their implementation.
